snuba/datasets/storages/factory.py

```diff
@@ -111,6 +113,11 @@
     **NON_WRITABLE_STORAGES,
 }
 
+STORAGES_WITH_DLQ: Mapping[StorageSetKey, Topic] = {
+    metrics_polymorphic_storage.get_storage_set_key(): Topic.DEAD_LETTER_METRICS,
+    sessions_raw_storage.get_storage_set_key(): Topic.DEAD_LETTER_SESSIONS,
+}
+
```

Comment:
For our other Kafka topics, the topic is generally a property of the storage / stream loader definition. I don't think we should start a new convention here.